charts/karpenter-crd/templates/karpenter.azure.com_aksnodeclasses.yaml

```diff
@@ -3,7 +3,7 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.17.2
+    controller-gen.kubebuilder.io/version: v0.17.3
```

Comment:
I assume this should go away when your other PR merges?

Reply:
Yes, though I thought this one will merge first. :-) Otherwise will just close